uisdk/src/main/res/layout/uisdk_view_quotes_item_prebook.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginBottom="@dimen/kh_uisdk_spacing_small"
+    android:background="@drawable/kh_uisdk_quote_list_item_background"
+    android:orientation="vertical"
+    android:padding="@dimen/kh_uisdk_spacing_none">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="?android:attr/selectableItemBackground"
+        android:orientation="vertical">
+
+        <LinearLayout
```

Comment:
why do we need this extra linear layout if it only has one child? can we add margin to its child and remove this extra layer?